runtimes/pythonrt/tests/loader_test.py

```diff
@@ -72,23 +72,6 @@ def action_fn(*args, **kw):
     ak_runner.load_code(root_path, action_fn, "simple")
 
 
-name_of_cases = [
```

Comment:
just to ensure - you don't need this test, since this covered in test_transform, right?

Reply:
Actually no, `name_of` is shown only in logging. However the new version returns the code `as-is` and I thought the test was redundant.